.azure/app-cloud-e2e.yml

```diff
@@ -31,7 +31,7 @@ pr:
       - ".azure/app-cloud-e2e.yml"
       - "src/lightning_app/**"
       - "requirements/app/**"
-      - "tests/integrations_app_examples/**"
+      - "tests/integrations_app/**"
```

Comment:
These changes are in master. A merge must've gone wrong. Can you rebase the PR?

Reply:
rebased :)